src/components/m-button/index.scss

```diff
@@ -10,7 +10,11 @@
   @include button--color();
 }
 
-.m-button__icon {
+// a-icon__svg is needed as css ordering is not loaded in the right order
+// I.e sizes in m-button__icon are overridden without this specifier.
+// Relates and fixes gh #836
+.m-button__icon,
+.m-button__icon.a-icon__svg {
```

Comment:
@LucaMele 
@jackblackCH 
Indeed, order is an issue, actually dependencies between components aren't expressed yet - tracked here #723 and a PR here #724
Unfortunately this workaround, does not fix the underlying issue, but rather creates additional bugs - reported so far: #836, #818 and #814 
Also all CSS-only consumers have to deal with it

Reply:
Are there any other components affected?

Reply:
So far I think, no.

Reply:
Agree with @AndyOGo, this way is not optimal. We will fix it in V2. For v1, i would say lets go with the current change and merge it.

Reply:
Unfortunately I found 2 more related issues 😞 
#841 #842